FreeRTOS/FreeRTOS-Kernel
# Allow xPortIsAuthorizedToAccessBuffer() API call only from mpu_wrappers_v2

Description
-----------
`xPortIsAuthorizedToAccessBuffer()` API is used only in the latest version of secure MPU wrappers ( V10.6.x ) , and not in the older wrapper versions. `xSchedulerRunning` variable is defined only in the latest version, hence when trying to compile the API with the old wrapper leads to compilation error. Hence , this PR adds the requirement of setting configUSE_MPU_WRAPPERS_V1 = 0 , to enable application writers to use this API.

Test Steps
-----------
MPU soak test on STM32H743ZI

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
